content/v2.0/get-started.md

```diff
@@ -194,6 +195,100 @@ docker exec -it influxdb /bin/bash
 {{% /tab-content %}}
 <!--------------------------------- END Docker -------------------------------->
 
+<!-------------------------------- BEGIN kubernetes---------------------------->
+{{% tab-content %}}
+
+You can install InfluxDB in a local kubernetes environment with Minikube.
+
+1. Install minikube.
+
+    On macOS, run:
+
+    ```
+    brew install minikube
+    ```
+
+2. Start minikube:
+
+    ```
+    minikube start
+    ```
+
+3. Save the following YAML configuration file somewhere on your machine:
+
+    ```yaml
+    kind: Namespace
+    apiVersion: v1
+    metadata:
+      name: influxdb
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: influxdb
+      namespace: influxdb
+    spec:
+      selector:
+        matchLabels:
+          app: influxdb
+      template:
+        metadata:
+          labels:
+            app: influxdb
+        spec:
+          containers:
+            - name: influxdb
+              image: quay.io/influxdb/influxdb:2.0.0-alpha
+              ports:
+                - containerPort: 9999
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: influxdb
+      namespace: influxdb
+    spec:
+      type: ClusterIP
+      ports:
+        - port: 9999
+          protocol: TCP
+          name: http
+      selector:
+        app: influxdb
+    ```
+
+4. Apply the configuration by running:
+
+    ```
+    kubectl apply -f <path-to-config>.yaml
+    ```
+
+    You should see an output like this:
+
+    ```
+    namespace/influxdb configured
+    deployment.apps/influxdb configured
+    service/influxdb configured
+    ```
+
+5. Ensure the service is running:
+
+    ```
+    $ kubectl get service -n influxdb
+    NAME       TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+    influxdb   NodePort   10.100.164.55   <none>        9999:31111/TCP   26m
+    ```
+
+6. Forward the port:
```

Comment:
This should be included in the verify step. First you check to make sure the services have started, then you forward the port so you can use the UI.